doc/introduction/circuits.rst

```diff
@@ -94,37 +85,73 @@ instantiated using the :func:`device <pennylane.device>` loader.
 
     dev = qml.device('default.qubit', wires=2, shots=1000)
 
-PennyLane offers some basic devices such as the ``'default.qubit'`` and ``'default.gaussian'``
-simulators; additional devices can be installed as plugins (see
+PennyLane offers some basic devices such as the ``'default.qubit'``, ``'default.mixed'``
+and ``'default.gaussian'`` simulators; additional devices can be installed as plugins (see
 `available plugins <https://pennylane.ai/plugins.html>`_ for more details). Note that the
 choice of a device significantly determines the speed of your computation, as well as
 the available options that can be passed to the device loader.
 
+.. note::
+
+    For example, check out the ``'lightning.qubit'`` `plugin <https://github.com/PennyLaneAI/pennylane-lightning>`_,
+    which is a fast state-vector simulator written in C++.
```

Comment:
this is now a hard dependency if you install PL, so it comes across for free! I suggest maybe changing this to lightning.gpu?
```suggestion
.. note::

    For example, check out the ``'lightning.gpu'`` `plugin <https://github.com/PennyLaneAI/pennylane-lightning-gpu>`_,
    which is a fast state-vector simulator supporting GPUs.
```